drivers/regulator/Kconfig.da1469x

```
@@ -0,0 +1,19 @@
+# Copyright 2023 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+config REGULATOR_DA1469X
+	bool "DA1469X regulators driver"
+	default y
+	depends on DT_HAS_RENESAS_SMARTBOND_REGULATOR_ENABLED
+	help
+	  Enable support for the Smartbond DA1469x regulators.
+
+if REGULATOR_DA1469X
+
+config REGULATOR_DA1469X_INIT_PRIORITY
+	int "Renesas DA1469x regulators driver init priority"
+	default KERNEL_INIT_PRIORITY_DEVICE
+	help
+	  Init priority for the Renesas DA1469x regulators driver.
+
+endif # REGULATOR_DA1469X
```

Single configs should use depends
```suggestion
config REGULATOR_DA1469X_INIT_PRIORITY
	int "Renesas DA1469x regulators driver init priority"
	default KERNEL_INIT_PRIORITY_DEVICE
	depends on REGULATOR_DA1469X
	help
	  Init priority for the Renesas DA1469x regulators driver.
```

---

@aasinclair thanks for a hint, change applied as suggested